src/api/public/apidocs-new/paths/source_project_name_cmd_addchannels.yaml

```diff
@@ -0,0 +1,16 @@
+post:
+  summary: Add channels to a source
+  description: Add channels to a source
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      $ref: '../components/responses/unknown_project.yaml'
+  tags:
+    - Sources - Projects
```

Comment:
The tag `Sources - Projects` is missing in the list of tags in the `OBS-v2.10.50.yaml` file. We should only add a `Sources - Projects` entry in that list.

Reply:
I've added the tag to the `OBS-v2.10.50.yaml` file. Let me know if it's ok now.

Reply:
Now it's added. Fine. Just only, insert the entry so the list keeps the alphabeticall order.